src/components/FullscreenLoadingIndicator.tsx

```diff
@@ -15,22 +19,69 @@ type FullScreenLoadingIndicatorProps = {
     /** Size of the icon */
     iconSize?: FullScreenLoadingIndicatorIconSize;
 
+    /** Timeout for the activity indicator after which we fire a log about abnormally long loading */
+    timeout?: number;
+
     /** The ID of the test to be used for testing */
     testID?: string;
 
     /** Extra loading context to be passed to the logAppStateOnLongLoading function */
     extraLoadingContext?: ExtraLoadingContext;
 };
 
-function FullScreenLoadingIndicator({style, iconSize = CONST.ACTIVITY_INDICATOR_SIZE.LARGE, testID = '', extraLoadingContext}: FullScreenLoadingIndicatorProps) {
+function FullScreenLoadingIndicator({
+    style,
+    iconSize = CONST.ACTIVITY_INDICATOR_SIZE.LARGE,
+    timeout = CONST.TIMING.ACTIVITY_INDICATOR_TIMEOUT,
```

Comment:
Do we need that prop? It's not used anywhere (at least in this PR) so I'd avoid adding it and make it configurable if (or when) we really need it

Reply:
Ahh yeah all props should be used. I must have missed that.